plugins/linuxplugin/ifplugin/interface_config.go

```diff
@@ -42,8 +42,10 @@ import (
 
 /* how often in seconds to refresh the microservice label -> docker container PID map */
 const (
-	dockerRefreshPeriod = 3 * time.Second
-	vethConfigNamespace = "veth-cfg-ns"
+	dockerRefreshPeriod     = 3 * time.Second
+	vethRefreshPeriod       = 50 * time.Millisecond
+	vethRefreshAttemptCount = 10
+	vethConfigNamespace     = "veth-cfg-ns"
 )
```

Comment:
Please define 3, 10, 50, ... as constants. sometime down the road we may also want to make them configurable.

Reply:
if we found this solution working, it will be configurable